### ISSUE TITLE
Problem with automatic gitolite-admin repo update

### ISSUE THREAD

**Issue by niosHD** [kubitron/redmine_git_hosting#83]
As a work around for the other bug [1] I tried to create a new project with "Automatically Initialize Git Repositories For New Projects = Enabled". At the first glance everything looked good in redmine and in the log. Unfortunately it is not working either. I had to manually resync the gitolite config with 

```
rake redmine_git_hosting:update_repositories RAILS_ENV=production
```

to get the requested repo.

```
Performing manual UpdateRepositories from command line (via rake at 2012-10-15 20:09:08)
Executing RESYNC_ALL operation on gitolite configuration
Fetching changes from gitolite-admin repository to /tmp/redmine_git_hosting/gitolite/gitolite-admin
Creating MD5 digests for Redmine Git Hosting hook
Digest for post-receive.redmine_gitolite.rb: c9a05723962684b6af4f9b27ca055f71
Our hook is already installed
Creating new entry 'test3' in gitolite.conf
  Letting gitolite create empty repository: 'repositories/test3.git'
Committing corrections to gitolite-admin repository
Updating Hook URL: [...]
Setting up hooks for repository 'test3' (in gitolite repository at 'repositories/test3.git')
```

After additional tests I found out that it is even worse. The automatic gitolite-admin repo update is not working for me any more.

I already tried to regenerate the scripts with

```
rake redmine_git_hosting:install_scripts RAILS_ENV=production WEB_USER=www-data
```

without notable improvements. (The file owner is ok and the the scripts are executable.)

What bothers me the most is that there are no errors reported at all. The repository is simply not updated ...

Any ideas on what I have missed here?

[1] https://github.com/kubitron/redmine_git_hosting/issues/82

**Comment by kubitron** [comment]
Can you see if t his problem goes away with the patch for #82?  (grab head of master).

Also, can you tell me what you mean by "the repository is not updated"?  Do you mean that you don't see anything from the repository link on the GUI?

**Comment by niosHD** [comment]
Well actually I meant that the changes from redmine were not propagated into the gitolite-admin repo. (e.g. creating new repos or adding new useres/keys to existing ones)

However, it seems like the problems where related after all. After upgrading to the current master (with the fix for #82) everything seems to work now.

Sorry for the false alarm ...

**Comment by kubitron** [comment]
Glad that it is working now.
